main.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const electron = require('electron');
+const ipc = electron.ipcMain;
+const app = electron.app;  // Module to control application life.
```

Comment:
Use `const app = require('app');` instead.
Same for all other `electron.` statements.